integration/mesos/pom.xml

```diff
@@ -58,6 +58,20 @@
       <artifactId>alluxio-underfs-hdfs</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.beust</groupId>
```

Comment:
This has no dependencies. I chose it over apache commons cli for its simplicity.